spec/presenters/grouped_result_set_presenter_spec.rb

```diff
@@ -165,6 +165,19 @@
     )
   end
 
+  let(:activity_facet) do
+    double(
+      OptionSelectFacet,
+      key: 'business_activity',
+      allowed_values: [
+        { 'value' => 'products-or-goods', 'label' => 'Products or goods' },
+        { 'value' => 'buying', 'label' => 'Buying' },
+      ],
+      value: %W(aerospace agriculture),
+      labels: %W(aerospace agriculture),
```

Comment:
Should the value and labels here match the allowed_values?

Reply:
They don't seem to have any effect in this test so I'll remove them from both facets.